docs/error-reporting.md

```diff
@@ -0,0 +1,30 @@
+# Error Reporting Explained
+
+## What's going on?
+
+The Lighthouse team is constantly trying to improve the reliability of our tools. We've added functionality to anonymously report runtime exceptions, given your consent, using [Sentry](https://sentry.io/welcome/). We will use this information to detect new bugs and avoid regressions.
```

Comment:
maybe
`"If you give your consent, we've added functionality to anonymously report runtime exceptions using [Sentry](https://sentry.io/welcome/)."`

Reply:
heh, but the functionality is there if they give consent or not, the phrase is modifying "report" ;)

rephrased